src/lib_ccx/params.c

```diff
@@ -1199,18 +1199,15 @@ int parse_parameters (struct ccx_s_options *opt, int argc, char *argv[])
 		{
 			int rc;
 			if (argv[i][strlen(argv[i])-1] != '+')
-			{
-				rc = append_file_to_queue (opt, argv[i]);
-			}
+				rc = append_file_to_queue(opt, argv[i]);
```

Comment:
Let's leave the brackets... 
While in other parts of the code they're less important, this specific file is too long and prone to errors, that's why we have these syntactically unneeded brackets here and there :-)